modules/administration-guide/pages/installing-che-on-openshift-using-the-web-console.adoc

```diff
@@ -27,7 +27,18 @@ Error from server (NotFound): customresourcedefinitions.apiextensions.k8s.io "ch
 
 . Install the {prod} Operator. See link:https://docs.openshift.com/container-platform/{ocp4-ver}/operators/admin/olm-adding-operators-to-cluster.html#olm-installing-from-operatorhub-using-web-console_olm-adding-operators-to-a-cluster[Installing from OperatorHub using the web console].
 
-. Create a {prod-short} instance from the {prod} Operator. See https://docs.openshift.com/container-platform/4.10/operators/user/olm-creating-apps-from-installed-operators.html[Creating applications from installed Operators].
+. Create the `{prod-namespace}` project in OpenShift as follows:
++
+[subs="+attributes"]
+----
+oc create namespace {prod-namespace}
+----
```

Comment:
That's completely ok for me.
Just one note, since the project name does n't start from `openshift-`, it is possible to create it via OpenShift UI without using `oc` tool

Reply:
Thanks for pointing this out, Anatolii. Fabrice's advice to me on this was to unify as much of the docs as possible so that we don't run into outdated downstream content issues in the future.